.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Exit if not on master branch
```

Comment:
I think you can remove this step.

Reply:
Ah yup, you're right.  I think on some of the other repos we may be able to combine "on push tags + "on push branches master" instead of having these as well?